common/reviews/api/rush-lib.api.md

```diff
@@ -1078,18 +1076,25 @@ export class RushConfiguration {
     readonly experimentsConfiguration: ExperimentsConfiguration;
     findProjectByShorthandName(shorthandProjectName: string): RushConfigurationProject | undefined;
     findProjectByTempName(tempProjectName: string): RushConfigurationProject | undefined;
-    getCommittedShrinkwrapFilename(variant?: string | undefined): string;
-    getCommonVersions(variant?: string | undefined): CommonVersionsConfiguration;
-    getCommonVersionsFilePath(variant?: string | undefined): string;
+    getCommittedShrinkwrapFilename(subspaceName?: string | undefined): string;
+    getCommonRushConfigFolder(subspaceName?: string | undefined): string;
+    getCommonTempFolder(subspaceName?: string | undefined): string;
+    getCommonVersions(subspaceName?: string | undefined): CommonVersionsConfiguration;
+    getCommonVersionsFilePath(subspaceName?: string | undefined): string;
     getImplicitlyPreferredVersions(variant?: string | undefined): Map<string, string>;
-    getPnpmfilePath(variant?: string | undefined): string;
+    getPackageManagerToolFilename(subspaceName?: string | undefined): string;
```

Comment:
I don't think we need this API.  The package manager choice (`pnpm`) and its version should be the same for all subspaces.

Reply:
Fixed in eff28adbeda17fa206bb21d37666212e4168d758